sys/openbsd/init.go

```diff
@@ -91,6 +93,13 @@ func (arch *arch) SanitizeCall(c *prog.Call) {
 		for _, f := range badflags {
 			flags.Val &= ^f
 		}
+	case "ioctl":
+		// Performing the following ioctl on a /dev/pf file descriptor
+		// causes the ssh VM connection to die.
+		request := c.Args[1].(*prog.ConstArg)
+		if request.Val == arch.DIOCKILLSTATES {
+			request.Val = 0
```

Comment:
Is 0 a magic value for ioctl or syzkaller? I didn't find any description of 0 in the ioctl man page.